Example/Example Swift/ComponentAPICoordinator.swift

```diff
@@ -682,9 +681,9 @@ extension ComponentAPICoordinator {
 
 extension ComponentAPICoordinator {
     
-    fileprivate func handleAnalysis(with extractions: [String: GINIExtraction]) {
+    fileprivate func handleAnalysis(with extractions: [Extraction]) {
         let payFive = ["paymentReference", "iban", "bic", "paymentReference", "amountToPay"]
-        let hasPayFive = extractions.filter { payFive.contains($0.0) }.count > 0
+        let hasPayFive = extractions.filter { payFive.contains($0.name ?? "no-name") }.count > 0
```

Comment:
So basically, if name is null, you compare against "no-name" which is guaranteed to fail? It's a little confusing. I'd rather write

`$0.name ? payFive.contains($0.name) : false`

It's still not super concise but it's easier to follow as logic.